src/GraphQL.Tests/Bugs/BugWithCustomScalarsInDirective.cs

```diff
@@ -1,11 +1,12 @@
+using GraphQL.DI;
 using GraphQL.Types;
 using GraphQLParser.AST;
 
 namespace GraphQL.Tests.Bugs
 {
-    public class BugWithCustomScalarsInDirective : QueryTestBase<BugWithCustomScalarsInDirectiveSchema, MsDiContainer>
+    public class BugWithCustomScalarsInDirective : QueryTestBase<BugWithCustomScalarsInDirectiveSchema>
     {
-        public BugWithCustomScalarsInDirective()
+        public override void RegisterServices(IServiceRegister Services)
```

Comment:
I use `Services` instead of `services`/ `register` here to minimize diff. @Shane32 so I would like to rename it after your review.

Reply:
Understood